### DISCUSSION TITLE
Error 'Fugashi'

### DISCUSSION THREAD

**Discussion by od3zza** [ogkalu2/comic-translate#57] [Q&A]
Hi, folks! When I try to install I'm facing this error:

_Using cached wrapt-1.16.0-cp310-cp310-win_amd64.whl (37 kB)
Building wheels for collected packages: fugashi
  Building wheel for fugashi (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [5 lines of output]
      running bdist_wheel
      running build
      running build_ext
      building 'fugashi' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for fugashi
  Running setup.py clean for fugashi
Failed to build fugashi
ERROR: Could not build wheels for fugashi, which is required to install pyproject.toml-based projects_

Someone knows how fix it?

**Comment by ogkalu2** [comment]
Open requirements.txt and remove the last 3 lines. Then try again
